uInverter pcb review

nvd21 opened the pull request:
# uInverter Schematic review
This PR is for the pcb review of the uInverter board.

Following are the details of the board -

Question |	Answer|
| --- | ---- |
Board Size (H x W) |	3.6" x 6.3"  (91.4mmX160mm)
Number of Layers |	2
Number of Components |	55
Pad via/ holes | 188
Number of Slots/Cutouts |	0
Silkscreen	| One side






## uInverter board routing snips -  
### Top layer

![pcbtop02](https://user-images.githubusercontent.com/78701509/116604473-cb85e080-a8f3-11eb-8202-c6d0a6d1440e.PNG)
  

![pcbtop01](https://user-images.githubusercontent.com/78701509/116604455-c4f76900-a8f3-11eb-816f-6d20052378a5.PNG)


### Bottom layer

![pcbbot02](https://user-images.githubusercontent.com/78701509/116604770-2d464a80-a8f4-11eb-8619-c4288ac532f1.PNG)


![pcbbot01](https://user-images.githubusercontent.com/78701509/116604514-d8a2cf80-a8f3-11eb-9ae6-114b81ee6253.PNG)


### Assembled board

![Capture](https://user-images.githubusercontent.com/78701509/116604549-e48e9180-a8f3-11eb-90e7-5c4d53307de9.JPG)


## Below are links for - 

- [BOM](https://github.com/Severson-Group/AMDC-Hardware/blob/uinv-rev-a-pcb/Accessories/ExpansionBoard_uInverter/REV20210506A/Prj_uinv-rev-a.csv)
- [Schematics](https://github.com/Severson-Group/AMDC-Hardware/blob/uinv-rev-a-pcb/Accessories/ExpansionBoard_uInverter/REV20210506A/uInverter-rev-a.pdf)

npetersen2 reviewed (changes requested):
@nvd21 Please go through and remove all unneeded files from the `Files changed` view on GitHub. Check out the other hardware repos in the group for hints on what should be included (e.g. not the `History/` folder from Altium)

elsevers reviewed (changes requested):
List compiled from design review meeting:

- [x] Repository file structure needs to be revised to match the expected structure outlined [here](https://github.com/Severson-Group/KnowledgeBase/blob/master/PCB-Design-Guidelines.md#repo-set-up). Also see other boards designs in the AMDC-Hardware repo as examples of the expected structure.
- [ ] Review the analog input signal chain of the AMDC and determine if any protection needs to be designed into the uInverter--either to protect the uInverter signal amplifiers (too much current draw on the amplifier IC) or to prevent damage on the AMDC (too much voltage to the AMDC)
- [ ] Add test points on "anything that moves" -- i.e., low voltage PWM signals, neutral point, current sense input (between each current sense resistor and inductor), phase outputs, DC bus, current sense outputs, DC bus sensed voltage
- [ ] Add ground vias to the decoupling capacitors

Overall, looks good!

npetersen2 commented:
@nvd21 can you also update the silkscreen ratings to be more clear as to what they are talking about?

![image](https://user-images.githubusercontent.com/20168990/116726260-a606de80-a9a8-11eb-81b7-82b7affad0e9.png)

- 12V DC makes sense.. if space allows, also add absolute maximum/minimum voltages.
- Make 4Arms max more obvious this is the PHASE currents, not DC bus.
- Provide rated power input for DC supply. At full RL load currents / freq, what is the expected power draw in watts? This will ensure the user uses the correct DC wall supply

npetersen2 commented:
Also, provide a small silkscreen label indicating barrel jack polarity -- are you assuming center is positive? The power supplies can come in either center positive or negative.

npetersen2 commented:
Minor note: your silkscreen fonts are not the same... some are "Arial" and some are the normal altium vector font. Please make them consistent.

npetersen2 commented:
Regarding test points, can you also add VDC test points **at the gate drivers**? I am a bit worried about no local decoupling caps at each gate driver, and a very chopped up 12V plane. I think the bus voltage at the gate drivers will not be very constant... would be nice to be able to probe this easily.

nvd21 commented:
Updates - 



> @nvd21 Please go through and remove all unneeded files from the `Files changed` view on GitHub. Check out the other hardware repos in the group for hints on what should be included (e.g. not the `History/` folder from Altium)

> List compiled from design review meeting:
> 
> * [x]  Repository file structure needs to be revised to match the expected structure outlined [here](https://github.com/Severson-Group/KnowledgeBase/blob/master/PCB-Design-Guidelines.md#repo-set-up). Also see other boards designs in the AMDC-Hardware repo as examples of the expected structure.
> * [x]  Review the analog input signal chain of the AMDC and determine if any protection needs to be designed into the uInverter--either to protect the uInverter signal amplifiers (too much current draw on the amplifier IC) or to prevent damage on the AMDC (too much voltage to the AMDC)
> * [x]  Add test points on "anything that moves" -- i.e., low voltage PWM signals, neutral point, current sense input (between each current sense resistor and inductor), phase outputs, DC bus, current sense outputs, DC bus sensed voltage
> * [x]  Add ground vias to the decoupling capacitors
> 
> Overall, looks good!


- Repository structure changed and unnecessary files removed.   
- Analog signal-chain reviewed (shouldn't be an issue).
-  Added lots of test points covering 'everything that moves'.  
- Added ground vias for all caps and wherever needed. 


> @nvd21 can you also update the silkscreen ratings to be more clear as to what they are talking about?
> 
> ![image](https://user-images.githubusercontent.com/20168990/116726260-a606de80-a9a8-11eb-81b7-82b7affad0e9.png)
> 
> * 12V DC makes sense.. if space allows, also add absolute maximum/minimum voltages.
> * Make 4Arms max more obvious this is the PHASE currents, not DC bus.
> * Provide rated power input for DC supply. At full RL load currents / freq, what is the expected power draw in watts? This will ensure the user uses the correct DC wall supply



- Added nominal and max rating on silkscreen.

> Also, provide a small silkscreen label indicating barrel jack polarity -- are you assuming center is positive? The power supplies can come in either center positive or negative.

- Added barrel-jack polarity marking on silkscreen.
- Also an example 12V Wall wart supply matching the connector on board mentioned in the documentation.


> Minor note: your silkscreen fonts are not the same... some are "Arial" and some are the normal altium vector font. Please make them consistent.

- Font is same throughout the board on silkscreen, except for "Severson Group" (which is default as on other boards).


> Regarding test points, can you also add VDC test points **at the gate drivers**? I am a bit worried about no local decoupling caps at each gate driver, and a very chopped up 12V plane. I think the bus voltage at the gate drivers will not be very constant... would be nice to be able to probe this easily.
- Decoupling caps are present at the Gate drivers already.


![pcbtop01](https://user-images.githubusercontent.com/78701509/117233437-c12e7f80-ade8-11eb-9e20-6adce566c68d.PNG)


![pcbtop02](https://user-images.githubusercontent.com/78701509/117233452-c55a9d00-ade8-11eb-8758-c57209d7bbeb.PNG)


![sc_calc](https://user-images.githubusercontent.com/78701509/117233496-d7d4d680-ade8-11eb-876b-63aecc5b63ca.JPG)



![uInverter3D_02](https://user-images.githubusercontent.com/78701509/117233461-c986ba80-ade8-11eb-8d03-2460f6667025.JPG)

GnomeTek reviewed (approved):
I see all issues as being addressed. I approve.

npetersen2 reviewed (changes requested):
@nvd21 please reduce the `Files changed` tab to less than 394 files. I will not actually review until this has been addressed.

npetersen2 commented on Accessories/ExpansionBoard_uInverter/REV20210506A/Prj_uinv-rev-a.csv:
```diff
@@ -0,0 +1,21 @@
+"HS380-ND","3","530002B02500G","Heat Sink Passive TO-220 Radial Thru-Hole 2.6C/W Black Anodized","HS1, HS2, HS3","530002","530002B02500G"
+"277-1667-ND","7","1935161","Conn Terminal Blocks 2 POS 5mm Solder ST Thru-Hole 16A","J1, J2, J3, J4, J5, J6, J7","PHOENIX_1935161","1935161"
+"553-3011-1-ND","3","100u","FIXED IND 100UH 5.3A 118 MOHM","L1, L2, L3","FP-PA4344_104NLT-MFG","PA4344.104NLT"
+"","10","MOUNT_HOLE, TitleBlockWEMPEC","MOUNTING HOLE, [NoValue]","MH1, MH2, MH3, MH4, MH5, TitleBlockWEMPEC1, TitleBlockWEMPEC2_U_Half_bridge_load1, TitleBlockWEMPEC2_U_Half_bridge_load2, TitleBlockWEMPEC2_U_Half_bridge_load3, TitleBlockWEMPEC3","MOUNTING_HOLE, [NoValue]","MOUNT_HOLE, TitleBlockWEMPEC"
```
In altium, change the type of the `MOUNTING_HOLE` and `TitleBlock` symbols to be either `Graphical` or `Mechanical` to not have them automatically included in the BOM.

Please make this change in altium, then regenerate the BOM file to ensure it is clean.

nvd21 replied:
Changes made in Altium file.

npetersen2 commented on Accessories/ExpansionBoard_uInverter/REV20210506A/README.md:
```diff
@@ -0,0 +1,11 @@
+# uInverter Board
```
This file should include a table of the PCB properties for ordering the revision. See the other hardware repos for examples. Examples: board size, layers, min clearance, etc.

nvd21 replied:
Included the table of PCB relevant details for PCB ordering.

npetersen2 commented on Accessories/ExpansionBoard_uInverter/docs/uInverter_connections.md:
```diff
@@ -0,0 +1,45 @@
+# uInverter Board
+
+This document decribes the details of the connections on the uInverter board. For detailed review of the connections, please refer [schematics](https://github.com/Severson-Group/AMDC-Hardware/tree/uinv-rev-a-pcb/Accessories/ExpansionBoard_uInverter/REV20210506A/uInverter-rev-a.pdf).
```
Provide relative links -- this link won't work once we merge this PR.

nvd21 replied:
Done!

npetersen2 commented on Accessories/TestBoard/altium/PCB.PcbDoc.htm:
```diff
@@ -0,0 +1,155 @@
+<html>
```
This file is not needed, please remove.

nvd21 replied:
Removed!

npetersen2 reviewed (changes requested):
Looking better, see my inline comments.

Also, please remove the gerber zip file -- we only include the text file gerbers in the `gerber/` folder, not the zip file.

npetersen2 commented on Accessories/ExpansionBoard_uInverter/README.md:
```diff
@@ -0,0 +1,14 @@
+- BNC connector to directly measure the current waveforms on an oscilloscope conveniently.    
+
+
+![PCB 3D](uInverter3D_02.JPG)
```
Can you move this jpg file into the `docs/images/` folder along with the other images? This cleans up the root folder.

nvd21 replied:
Updates: 
- Added more explanation on the `R/L ratio` and `Ripple Current` section and how these aspects are considered in design. 
- Also mentioned the emulation of small inductance high speed motors. 
- Removed the image in the root folder and use the one in `REVXXXX`

npetersen2 reviewed (changes requested):
Almost there -- a few more comments inline.

**Comments about `docs/README.md` file:**

I am liking your main `docs/README.md` file for the most part. I like how you outline the design procedure and considerations for each component in the system. Very nice.

However, you do not show that the selected components meet the requirements -- specifically talking about the R and L selection. For example, what is the R/L ratio for the uInverter board? Also, why do you tell us the `Ripple Current` section, but then not relate it back to your design? I think the discussion would make more sense if you added more details relating your design back to the general design discussions. I think this is only a few sentences here and there.

Also, when discussing the goal for R and L, please make it clear that the target motor which you are emulating is specifically chosen to be a low-inductance, "high-speed" motor so that it matches the prototype machine designs from the group. Not all motors have 100uH inductance!

elsevers commented:
@nvd21 , if you feel you have addressed all of @npetersen2's requests, please re-request his review

elsevers commented on Accessories/ExpansionBoard_uInverter/REV20210506A/README.md:
```diff
@@ -0,0 +1,30 @@
+Documentation about the design of uInverter board is located in [docs folder.](../docs/)
+
+
+![PCB 3D](uInverter3D_02.jpg)
```
This link does not work.

nvd21 replied:
Updated! My apologies for this. (The .md reader in my PC renders incorrectly!).

elsevers commented on Accessories/ExpansionBoard_uInverter/REV20210506A/README.md:
```diff
@@ -0,0 +1,30 @@
+
+Please check the [docs folder](../docs/) for more details on the connectors and pin-outs.
+
+Documentation about the design of uInverter board is located in [docs folder.](../docs/)
```
Odd to see this same folder linked twice in two subsequent sections that are each their own paragraph.

nvd21 replied:
Updated.

elsevers reviewed (changes requested):
Thanks for your efforts on this Nikhil. Before I am comfortable approving this PR, I want to see revisions to your documentation. These revisions should focus on improving the quality of the writing (eliminating grammar and typesetting issues as well as writing in a more specific and clear manner and providing more context and narrative on the uInverter's approach and purpose), as I outline below. Please revise your presentation of material to be more consistent with our other documentation in the AMDC hardware repository. See [AMDC hardware docs](https://github.com/Severson-Group/AMDC-Hardware/tree/develop/docs), [DAC expansion docs](https://github.com/Severson-Group/AMDC-Hardware/tree/develop/Accessories/ExpansionBoard_DAC/docs), and [the new CAN board docs](https://github.com/Severson-Group/AMDC-Hardware/tree/can-rev-a-docs/Accessories/ExpansionBoard_CAN/docs) for examples of effective documentation and the style that we would like you to use.

# Feedback on `docs/readme.md`:

- [x] Are you able to provide a vectorized SVG for your block diagram? It also is not in the same style as we typically have for our block diagrams. See the hardware docs [here](https://github.com/Severson-Group/AMDC-Hardware/tree/develop/docs) and [here](https://github.com/Severson-Group/AMDC-Hardware/tree/develop/Accessories/ExpansionBoard_DAC/docs) for examples.

--------

- [x] As currently written, this sentence does not add any new information. It is also missing a `the`
![image](https://user-images.githubusercontent.com/7421348/117668188-51632080-b16b-11eb-8aa9-ae58cf02f894.png)

-----------

- [x] Errant mark?
![image](https://user-images.githubusercontent.com/7421348/117668266-6475f080-b16b-11eb-87c3-a0c7e534111a.png)

-----------

- [x] Are we missing a sentence here or something? It seems quite out of place to see this lone bullet. Where it the context?
![image](https://user-images.githubusercontent.com/7421348/117668355-79eb1a80-b16b-11eb-9fee-5b8af2cc6948.png)

-----------

- [x] Missing a `the` at the start of the first sentence. But the bigger point is that this paragraph is presenting information in the wrong order. We don't yet know that you are using a gate drive in place of a half-bridge. Explain that first! Then work down to the details

![image](https://user-images.githubusercontent.com/7421348/117668597-b4ed4e00-b16b-11eb-92b5-c790686c8895.png)

-----------

- [x] Again, we're missing the story here. We don't know how you are doing current sensing, but yet you are immediately telling us about properties of a components. You need to start with the narrative 
1. _What is the approach you are taking to sense current?_ Include a simple schematic figure if possible; 
2. _What requirements do you have for this and why?_ This should include discussion of what the AMDC interface requires, with links to supporting AMDC docs. For example, you say `an acceptable Common Mode voltage range of...`--how do we know this is acceptable? What determines this? (also, you should not be capitalizing `Common Mode`)
3. _Which components did you select and what nuances should we be aware of?_ At the end, you tell us that you are using the same difference amplifier as is on the AMDC, but you haven't explained that the `INA143U` is a difference amplifier. You should also be linking to the AMDC doc that describes using this difference amplifier.

![image](https://user-images.githubusercontent.com/7421348/117669362-6b513300-b16c-11eb-8e89-c62bcf1b3a6d.png)

- I have the same general comments about `Shunt Resistor`, `Load Inductor`, `DC link capacitor` sub sections. Consider adding a section before your `Requirements` section the discuss the functional approach taken for the board design. It is hard to understand requirements if we don't understand what you are trying to do. It would be worth spending some time reading our markdown docs of other board layouts.
--------

- [x] A link to `uInverter_connections.md` either needs to be provided in this file or the contents of  `uInverter_connections.md` need to be merged into this file.

-------
- [ ] Nice. This is useful to see summarized. However, I am not sure how to interpret `L` in this table. Can you use `R` and `L` in a voltage/current differential equation?
![image](https://user-images.githubusercontent.com/7421348/117672482-81142780-b16f-11eb-8d1f-94233ff295e9.png)

--------

# Feedback on `uInverter_connections.md`
- [ ] `Please refer schematics` is not grammatically correct
![image](https://user-images.githubusercontent.com/7421348/117671370-583f6280-b16e-11eb-8da7-13cc8606c36e.png)

--------

- [ ] Which connectors on the AMDC? Be specific. Give us a link to a markdown doc if possible
![image](https://user-images.githubusercontent.com/7421348/117671284-3fcf4800-b16e-11eb-8392-4482487bd452.png)

----------

- [ ] Instead of showing us a screenshot of a schematic, please provide a markdown table of connection signals to be consistent with the style we use in our other AMDC docs. See [here](https://github.com/Severson-Group/AMDC-Hardware/tree/develop/Accessories/ExpansionBoard_DAC/docs#amdc-connector) for an example. 
![image](https://user-images.githubusercontent.com/7421348/117671473-71481380-b16e-11eb-80e7-27f35d6b08bf.png)

![image](https://user-images.githubusercontent.com/7421348/117672219-414d4000-b16f-11eb-9a7f-b6ad89a2120e.png)

npetersen2 reviewed (changes requested):
Only minor requests:

1. Please rename the `Altium/` dir to be lower case to match other repos
2. Please rename the BOM file in the `REVxxxx` folder to have the text "BOM" somewhere in the filename so people know it is the BOM.
3. It is a bit odd to have the `snapshots` folder in the `REVxxxx` folder, but then have a JPG in the root level (not in the snapshot folder). Since you have the snapshot folder, can you put the extra JPG file into it? Remember to update the image link in the markdown files to match the new path.

Also, I see you have schematic screenshots in the `snapshot` folder. Why? Can you delete these?

npetersen2 commented:
After more thought, I'd like you to update your `docs/` block diagram a bit:

Overall, the purpose of the block diagram is provide insight into the circuit design quickly. The audience is both users of the board and future developers.

1. Please use the same fonts, colors, styles, etc as the other doc block diagrams from other board designs (see Eric's linked examples in his feedback). It is important that the docs are consistent across the whole AMDC Platform to create a cohesive experience for users, and to provide good examples for future developers on the project. Most of the other block diagrams have been created in the free Inkscape software. Typically, people download an existing block diagram svg file and simply modify it to fit their board design -- this keeps styling consistent.
2. The purpose of the block diagram is to provide a high-level story of the hardware design, **not to provide a copy of the Altium schematics**. Your block diagram seems to mostly be a copy of the Altium schematics. I realize that it is challenging to distill the hardware design into something easier to understand than the Altium schematics, but it is critical to help users understand the board quickly.

Some ideas for updating the block diagram: I think your block diagram feels like Altium schematics because it is has a lot of wires going everywhere, some even cross over text and labels. Also, there is a lot of replication (3x of everything). I don't think you should remove the duplication, but perhaps simplify the labeling so it is a bit clearer to look at.

See the block diagrams below for examples of good diagrams which show helpful information without simply copying the schematics. None of these block diagrams resemble a schematic, yet they convey lots of helpful information quickly.
- https://github.com/Severson-Group/AMDC-Hardware/blob/develop/docs/Analog.md
- https://github.com/Severson-Group/AMDC-Hardware/blob/develop/docs/Encoder.md
- https://github.com/Severson-Group/AMDC-Hardware/blob/develop/docs/PowerDistribution.md
- https://github.com/Severson-Group/AMDC-Hardware/blob/develop/docs/PowerStack.md

Your labeling of `I_A` ... `I_C` is a bit confusing -- typically that would be used to label actual currents flowing in a wire. However, you seem to be indicating it means a low-voltage signal corresponding to the flowing current.

You also do you show the location of the jumper connectors on your block diagram. These are critical to include to help users of the board determine their purpose.

npetersen2 reviewed (approved):
@nvd21 looks good -- thanks for making the updates.

@elsevers can you take a look?

elsevers reviewed (changes requested):
This is looking a lot better! A few more changes are still needed:

- [ ] Block diagram: increase spacing around `To AMDC` `From AMDC` so that the red hashed line does not touch the text letters
![image](https://user-images.githubusercontent.com/7421348/118498343-8c290380-b6eb-11eb-8c92-36ecf6afc172.png)

-----------

- [ ] Block diagram legend: `PWM signals` arrow does not appear to appear to be perfectly horizontal and it is not the same length as the `analog sensing signals` arrow. `analog sensing signals` should have a capital `A` to be consistent with other entries in the legend. `Jumper` appears to have a different font than the other legend entries. The text for all legend entries needs to be better aligned

![image](https://user-images.githubusercontent.com/7421348/118498639-ddd18e00-b6eb-11eb-86f5-b7632bf09b3b.png)

------
- [ ] Block diagram: the purple `PWM-A`, `PWM-B`, and `PWM-C` arrows seem to rely on lines which are of two different widths. The green arrows for `Ia_sense`, `Ib_sense`, `Ic_sense` have different horizontal lengths. Please make all this consistent.

![image](https://user-images.githubusercontent.com/7421348/118501033-f3e04e00-b6ed-11eb-8ae5-a15d96460ecf.png)

![image](https://user-images.githubusercontent.com/7421348/118501060-f9d62f00-b6ed-11eb-827e-8b7f33da332c.png)

---------

- [ ] `docs/README.md` structure is still not consistent with our other documentation. The very first sentences under the document title should be describing the purpose of this document, not the purpose of the uInverter. I suggest you add a `Purpose` subsection (like our [AMDC Test Board](https://github.com/Severson-Group/AMDC-Hardware/tree/develop/Accessories/TestBoard/docs#purpose) doc has) that describes the purpose of your uInverter. You are also missing the `Relevant Hardware Versions` section.

![image](https://user-images.githubusercontent.com/7421348/118498922-196c5800-b6ec-11eb-99fe-01c5139c2b82.png)

----------

- [ ] `docs/README.md Gate Driver` section, this comma should be removed. Also try to revise to eliminate use of the word `thus`. 

![image](https://user-images.githubusercontent.com/7421348/118501299-2e49eb00-b6ee-11eb-97e5-bb1813097284.png)

----------
- [ ] `docs/README.md Current Sense Amplifier` section: you are using `SNR` without defining it. I suggest that you write out the full phrase and do not use the acronym. Also, eliminate `also helps` from this sentence.

![image](https://user-images.githubusercontent.com/7421348/118501639-900a5500-b6ee-11eb-9e8e-2f6e72974022.png)


----------

 - [ ] `docs/README.md` frequently concludes a paragraph with `as follows -`. Instead of using `-` please use a colon `as follows:`
 
 ------
 
 - [ ] `docs/README.md` remove the hyphen in the `References -` section title 
 
![image](https://user-images.githubusercontent.com/7421348/118499510-a0213500-b6ec-11eb-8af1-cf672c868abc.png)

--------

 - [ ] `docs/README.md` Equations are currently being typeset as quote blocks (the `> ` character prefaces them). This is not consistent with our other docs. See [Analog.md](https://github.com/Severson-Group/AMDC-Hardware/blob/develop/docs/Analog.md) as an example. 
 
 _uInverter README.md does this:_
 
![image](https://user-images.githubusercontent.com/7421348/118499945-fee6ae80-b6ec-11eb-9f12-1a5d3a79baae.png)

_[Analog.md](https://github.com/Severson-Group/AMDC-Hardware/blob/develop/docs/Analog.md) does this:_
![image](https://user-images.githubusercontent.com/7421348/118500053-132aab80-b6ed-11eb-8948-6994aefef6de.png)

---------

- [ ] `docs/README.md`, There seems to be an extra `-` in the `Current Sense Amplifier` section
![image](https://user-images.githubusercontent.com/7421348/118500328-508f3900-b6ed-11eb-9490-d8799f7ce220.png)

----------

- [ ] `docs/README.md`: `Component selection` heading has capitalization which is inconsistent with the rest of the document. Is also has the same heading level as `Gate Driver`, `Current Sense Amplifier`, etc.. However, I think you intended these items to be subheadings. Please fix

![image](https://user-images.githubusercontent.com/7421348/118500605-8df3c680-b6ed-11eb-8929-a31aa94ab3f8.png)

npetersen2 commented:
@nvd21 In the docs, you mention that the absolute range for DC voltage input is 10V to 12V. Then, you say the gate driver ICs can accept up to ~30V.

Is there anything that will break if the board is used with larger DC voltage input, i.e. 30V? If not, can you update the table to give the nominal value of 12V as well as absolute min and max?

nvd21 commented:
> @nvd21 In the docs, you mention that the absolute range for DC voltage input is 10V to 12V. Then, you say the gate driver ICs can accept up to ~30V.
> 
> Is there anything that will break if the board is used with larger DC voltage input, i.e. 30V? If not, can you update the table to give the nominal value of 12V as well as absolute min and max?

@npetersen2 The maximum voltage is mentioned as 12V considering safety even in an accidental case of DC current flowing in the phase leg. If the applied voltage is higher than 12V, the current flowing in the phase leg in the accidental case of short circuit/single-phasing would damage the RL load. It would also push the current sense amplifier in saturation, which would exceed the acceptable limits of analog amplifiers on the AMDC board.

elsevers commented:
> > @nvd21 In the docs, you mention that the absolute range for DC voltage input is 10V to 12V. Then, you say the gate driver ICs can accept up to ~30V.
> > Is there anything that will break if the board is used with larger DC voltage input, i.e. 30V? If not, can you update the table to give the nominal value of 12V as well as absolute min and max?
> 
> @npetersen2 The maximum voltage is mentioned as 12V considering safety even in an accidental case of DC current flowing in the phase leg. If the applied voltage is higher than 12V, the current flowing in the phase leg in the accidental case of short circuit/single-phasing would damage the RL load. It would also push the current sense amplifier in saturation, which would exceed the acceptable limits of analog amplifiers on the AMDC board.

Thanks Nikhil. Nathan's request is for you to comment on if there is anything a user should be careful about if they try to push the input DC supply above 12V **in the markdown doc** -- could you do this?
